test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -276,6 +276,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.index_management.enableIndexStats (any)',
         'xpack.index_management.editableIndexSettings (any)',
         'xpack.index_management.enableDataStreamsStorageColumn (any)',
+        'xpack.index_management.enableMappingsSourceField (any)',
```

Comment:
Could we just move this down below the comment with the other conditional flags, or comment as 'any' due to offering based schema?

Reply:
Sure, I moved it below the comment as well as the rest of the index management offering-based configs for consistency.